build

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+
+for component in cassandra collector query web; do
+    pushd $component
+    docker build -t openzipkin/zipkin-${component}-k8s:1.9.0 .
```

Comment:
heh.. we're up to 1.16.0

Mind bumping this? would be neat to have this available for the tracing workshop tomorrow.